src/Http/Response.php

```diff
@@ -26,6 +26,11 @@ public function getHeaders(): array
         return $this->headers;
     }
 
+    public function getContentType(): string
+    {
+        return $this->headers['content-type'] ?? '';
```

Comment:
Wellicht een idee om ook te checken op 'Content-Type'? 
Ik wel eens iets moeten doen zoals:
`$contentType = $headers['content-type'] ?? $headers['Content-Type'] ?? '';`

Reply:
Ik weet niet hoe, maar die method is verdwenen.. Ik kan me ook niet herinneren dat ik die er in heb gezet 😬

Reply:
Dit lijkt op een andere branch toegevoegd te zijn. Zo te zien heb je zelf die change gemaakt https://github.com/OpenWebconcept/plugin-owc-gravityforms-zaaksysteem/pull/26/files#diff-f6919b0f7fddf8ecc9da906c1a52a42e724966f3965be170e0ad30de2f2ee786R31 ;-)